doc/sphinx/reactors.rst

```diff
@@ -195,3 +181,230 @@ temperature:
+
+3. *Optional:* Set up the boundary conditions and flow devices between reactors or reservoirs.
+
+4. Define a reactor network which contains all the reactors previously created.
+
+5. Advance the simulation in time, typically in a for- or while-loop. Note that only the current state is stored in Cantera by default. If you want to observe the transient states, you manually have to keep track of them.
+
+6. Analyze the data.
+
+Note that Cantera always solves a transient problem. If you are interested in steady-state conditions, you can run your simulation for a long time until the states are converged (see e.g. example `surf_pfr.py <cython/examples/reactors_surf_pfr.html>`_, `combustor.py <cython/examples/reactors_combustor.html>`_).
+
+Cantera comes with a broad variety of well-commented example scrips for reactor networks. Please refer to them for further information (`Python <cython/examples.html>`_, `Matlab <matlab/examples.html>`_).
+
+
+Common Reactor Types and their Implementation in Cantera
+========================================================
+
+
+Batch Reactor at Constant Volume or at Constant Pressure
+--------------------------------------------------------
+
+If you are interested in how a homogeneous chemical composition changes in time when it is left to its own, a simple batch reactor can be used. Two versions are commonly considered: A rigid vessel with fixed volume but variable pressure, or a system idealized at constant pressure but varying volume.
+
+In Cantera, such a simulation can be performed very easily. The initial state of the solution can be specified by composition and a set of thermodynamic parameters (like temperature and pressure) as a standard Cantera solution object. Upon its base, a general (Ideal Gas) Reactor or an (Ideal Gas) Constant Pressure Reactor can be created, depending on if a constant volume or constant pressure batch reactor should be considered, respectively. The behavior of the solution in time can be simulated as a very simple Reactor Network containing only the formerly created reactor.
+
+An example for such a Batch Reactor is `reactor1.py <cython/examples/reactors_reactor1.html>`_.
+
+
+Continuously Stirred Tank Reactor
+---------------------------------
+
+In literature, Continuously Stirred Tank Reactors (CSTR) are often used to perform basic computations like blowout limits in combustion systems. They are also often referred to as Well-Stirred Reactor (WSR), or Perfectly Stirred Reactor (PSR), or Longwell Reactor.
+
+A CSTR assumes a confined volume in which the fluid is homogeneous at all points. A stream of reactants enter the confinement at a steady mass flow rate `\dot{m}`. The reactor is assumed to be steady. Therefore, the identical mass flow rate `\dot{m}` of products have to exit the confinement. The mass contained in the confinement `m` divided by the mass flow rate `\dot{m}` defines the mean residence time of the fluid in the confinement.
```

Comment:
You're right, judging from the name CSTR is not necessarily steady-state, I was wrong with my formulation. What do you think about this:

    A Continuously Stirred Tank Reactor (CSTR), also often referred to as 
    Well-Stirred Reactor (WSR), Perfectly Stirred Reactor (PSR), or Longwell 
    Reactor, is essentially a single Cantera reactor with an inlet, an outlet, and 
    constant volume. Therefore, the `Governing Equations for Single Reactors`_ 
    defined above apply accordingly.

    Historically, CSTRs have often been considered to operate at steady state. In 
    this case, the mass flow rate `\dot{m}` is constant and equal at inlet and 
    outlet. The mass contained in the confinement `m` divided by `\dot{m}` defines 
    the mean residence time of the fluid in the confinement.

    At steady state, the time derivatives in the governing equations become zero, 
    and the system of ordinary differential equations can be reduced to a set of 
    coupled nonlinear algebraic equations. A Newton solver could be used to solve 
    this system of equations. However, a sophisticated implementation might be 
    required to account for the strong nonlinearities and the presence of multiple 
    solutions.

    Cantera does not have such a Newton solver implemented. Instead, steady CSTRs 
    are simulated by considering a time-dependent constant volume reactor with 
    specified in- and outflow conditions. Starting off at an initial solution, the 
    reactor network containing this reactor is advanced in time until the state of 
    the solution is converged. An example for this procedure is `combustor.py 
    <cython/examples/reactors_combustor.html>`_.

    A problem can be the ignition of a CSTR: If the reactants are not reactive 
    enough, the simulation can result in the trivial solution that inflow and 
    outflow states are identical. To solve this problem, either the reactor can be 
    initialized with a high temperature and/or radical concentration, or a reactive 
    species like atomic hydrogen can be temporarily injected to start the 
    reactions. See `combustor.py <cython/examples/reactors_combustor.html>`_ for an 
    example.